src/Features/LanguageServer/ProtocolUnitTests/Ordering/RequestOrderingTests.cs

```diff
@@ -132,7 +132,7 @@ public async Task LongRunningSynchronousNonMutatingTaskDoesNotBlockQueue()
 
             using var testLspServer = CreateTestLspServer("class C { }", out _);
 
-            // Cancel all requests if we hang for 5 seconds. This will result in a failed test run.
+            // Cancel all requests if the request queue is blocked for 1 minute. This will result in a failed test run.
```

Comment:
fwiw, @sharwell was opposing to a similar change in https://github.com/dotnet/roslyn/pull/53501.

Reply:
That's correct. The term 'hang' is explicitly allowed in source code, but the internal system still files bugs against us for each use. In this case, the proposed replacement does not impair clarity so it's fine to proceed.